src/Files.App/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -215,7 +215,7 @@
 
 					<VisualStateManager.VisualStateGroups>
 						<VisualStateGroup x:Name="CheckboxVisibilityStates">
-							<VisualState x:Name="Normal" />
+							<VisualState x:Name="HideCheckbox" />
```

Comment:
Can you do this for the tiles layout as well?

Reply:
Sure thing